python/paddle/__init__.py

```diff
@@ -24,8 +24,13 @@
 from .batch import batch  # noqa: F401
 from .fluid import monkey_patch_variable
 from .fluid.dygraph import monkey_patch_math_varbase
+from .fluid import core
+from paddle.fluid.framework import _test_eager_guard
```

Comment:
Add this to `__init__` may cause used end can see it, find another way maybe

Reply:
How about patch it in `_test_eager_guard` when we use `_test_eager_guard`

Reply:
done，thx！